.evergreen/.evg.yml

```diff
@@ -1081,6 +1101,12 @@ buildvariants:
   tasks:
     - name: ".ocsp"
 
+- matrix_name: "atlas-data-lake-test"
+  matrix_spec: { auth: "noauth", ssl: "ssl", jdk: "jdk11", version: "*", os: "ubuntu" }
```

Comment:
Why "noauth", since ADL requires authentication?

Reply:
Perhaps auth is just ignored entirely by `run-mongohouse-local.sh` so its value doesn't matter.  If so can it just be left out?

I suspect that this just shouldn't be a matrix at all, and should look more like `atlas-test` below.

Reply:
I have updated this to mimic `atlas-test`.